packages/react-tooltip/src/stories/TooltipCustomMount.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { Button } from '@fluentui/react-button';
+import { SlideTextRegular } from '@fluentui/react-icons';
+
+export const CustomMount = (props: Partial<TooltipProps>) => {
+  const [ref, setRef] = React.useState<HTMLElement | null>();
+
+  return (
+    <div ref={setRef}>
+      <Tooltip mountNode={ref} content="Example tooltip" relationship="label" {...props}>
+        <Button icon={<SlideTextRegular />} size="large" />
+      </Tooltip>
+    </div>
```

Comment:
```suggestion
    <>
      <Tooltip mountNode={ref} content="Example tooltip" relationship="label" {...props}>
        <Button icon={<SlideTextRegular />} size="large" />
      </Tooltip>
      <div ref={setRef} />
    </>
```

nit, I would rather place it into a separate element to make example clearer

Reply:
yeah, that makes it obvious that it is about 'targeting' a div, even if the div is somewhere totally different.